docs/creating-new/cmake-dependencies.rst

```diff
@@ -70,8 +70,11 @@ In terms of CMake code it means adding ``HunterGate`` and ``hunter_add_package``
   :align: center
 
 Note that the SHA1 of the Hunter archive in ``HunterGate`` commands of ``Bar`` and
-``Roo`` can be different. Same location will be set automatically internally
-you don't have to manage it manually.
+``Roo`` can be different. The main project may contain subprojects also using the
+``HunterGate`` command. The subprojects may be introduced as hunter dependencies,
+git submodules, or just the whole project added as subfolder.
```

Comment:
> The subprojects may be introduced as hunter dependencies

If we are calling "subprojects" the source subtree that have `project` command and added by `add_subdirectory` then Hunter dependencies are not subprojects.

> git submodules

This is a little bit confusing. CMake source tree and VCS are orthogonal. You can have subprojects without git submodules and submodules which is not used as a subprojects.